### ISSUE TITLE
Import/export of VRFs is missing fields

### ISSUE THREAD

**Issue by TheRealBecks** [nautobot/nautobot#1797]
### Environment
* Python version: 3.8.10
* Nautobot version: 1.3.4
* OS: Ubuntu 20.04

### Steps to Reproduce
Follow the standard installation process for an Ubuntu 20.04 server:
1. https://nautobot.readthedocs.io/en/stable/installation/ubuntu/
2. https://nautobot.readthedocs.io/en/stable/installation/nautobot/
3. https://nautobot.readthedocs.io/en/stable/installation/services/
4. https://nautobot.readthedocs.io/en/stable/installation/http-server/
5. Open the browser and check that Nautobot is working --> Yes, works!
6. Create a `route target` manually or import my example:

```
name,description,tenant
65500:110,VRF default,
```

7. Create a VRF manually via the `VRF` form:

```
Name: default
RD: 65500:110
Select 'Enforce unique space'
Import targets: Select '65500:110'
Export targets: Select '65500:110'
```

8. Go to the `VRF` page and click on the `Export` button, open the file in the editor --> `Route Targets` with `Import targets` and `Export targets` are missing in the export file

### Expected Behavior
Export: All configured fields get exported. 
Import: All fields can be imported.

### Observed Behavior
Export: Some configured fields are missing in the file.
Import: Some fields can't be imported, but manually configured in the form.

**Comment by TheRealBecks** [comment]
It looks like the import/export is still kind of broken?

**Comment by glennmatthews** [comment]
If I'm reading this right, it's another case of #254.

**Comment by glennmatthews** [comment]
Needs re-testing after #3722 and #3244 are completed.

**Comment by bryanculver** [comment]
@nautobot/core Is this addressed with #3722? VRF's serializer defines `__all__` fields so I think we're good here?

**Comment by bryanculver** [comment]
Validated fixed in 2.0.0b2 on https://next.demo.nautobot.com/ipam/vrfs/

Import targets and Export targets for example are contained. We have a separate issue around associations (particularly M2M) being addressed separately (see: https://github.com/nautobot/nautobot/issues/4103#issuecomment-1648297689)
